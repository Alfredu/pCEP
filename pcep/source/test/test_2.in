detect2.dolce
1 InputEvent int InputParameter 4
1 InputEvent int InputParameter 2
1 InputEvent int InputParameter 4
1 InputEvent int InputParameter 4
1 InputEvent int InputParameter 2
1 InputEvent int InputParameter 1
1 InputEvent int InputParameter 4
1 InputEvent int InputParameter 2

