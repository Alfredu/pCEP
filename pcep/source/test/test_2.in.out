2 OutputEventTimeWindow int theCount 1 int theGroup 4
2 OutputEventTupleWindow int theCount 1 int theGroup 4
2 OutputEventTimeWindow int theCount 1 int theGroup 2
2 OutputEventTupleWindow int theCount 1 int theGroup 2
2 OutputEventTimeWindow int theCount 2 int theGroup 4
2 OutputEventTupleWindow int theCount 2 int theGroup 4
2 OutputEventTimeWindow int theCount 3 int theGroup 4
2 OutputEventTupleWindow int theCount 3 int theGroup 4
2 OutputEventTimeWindow int theCount 2 int theGroup 2
2 OutputEventTupleWindow int theCount 2 int theGroup 2
2 OutputEventTimeWindow int theCount 1 int theGroup 1
2 OutputEventTupleWindow int theCount 1 int theGroup 1
2 OutputEventTimeWindow int theCount 4 int theGroup 4
2 OutputEventTupleWindow int theCount 2 int theGroup 4
2 OutputEventTimeWindow int theCount 3 int theGroup 2
2 OutputEventTupleWindow int theCount 2 int theGroup 2
7 Shutdown int Result 0 
